tools/projmgr/src/ProjMgrKernel.cpp

```diff
@@ -52,6 +52,18 @@ XMLTree* ProjMgrKernel::CreateXmlTree() const
 bool ProjMgrKernel::GetInstalledPacks(list<RtePackage*>& packs) {
   list<string> pdscFiles;
   RteFsUtils::GetPackageDescriptionFiles(pdscFiles, theProjMgrKernel->GetCmsisPackRoot(), 3);
+
+  // Find pdsc files from local repository
+  list<string> localPdscUrls;
+  if (!GetLocalPacksUrls(theProjMgrKernel->GetCmsisPackRoot(), localPdscUrls)) {
+    return false;
```

Comment:
Can we add a test case for this situation?

Reply:
I believe this situation will happen only if the local pack repository is malformed, but it can be emulated and tested.